app/main.py

```diff
@@ -1,7 +1,31 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+city = "Paris"
```

Comment:
According to the checklist, constants should be written in uppercase. Here, `city` is a constant and should therefore be written in uppercase, like `CITY`.